pkgs/desktops/gnome/games/gnome-2048/default.nix

```
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchurl
+, wrapGAppsHook
+, meson
+, vala
+, pkg-config
+, ninja
+, itstool
+, clutter-gtk
+, libgee
+, gnome
+, gtk3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-twenty-forty-eight";
+  version = "3.38.2";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/gnome-2048/${lib.versions.major version}/gnome-2048-${version}.tar.xz";
```

As it is right now it fails to fetch the package from the server... it should be:
`url = "mirror://gnome/sources/gnome-2048/${lib.versions.major version}.${lib.versions.minor version}/gnome-2048-${version}.tar.xz";`

---

After fixing the url, it builds and runs... so I'll approve after the suggested fix

---

... or just use `${lib.versions.majorMinor version}`

```suggestion
    url = "mirror://gnome/sources/gnome-2048/${lib.versions.majorMinor version}/gnome-2048-${version}.tar.xz";
```

---

Oops! I guess `nix-prefetch-url` prevented me from catching this mistake? For my future reference, is there a way to selectively un-prefetch a source so that my local testing would catch mistakes like this?

---

Try `nix-build -A gnome-2048.src --check` but not sure if it works on FODs too.

---

Yeah, that does catch it. Thanks!